cmake/libs/Libbacktrace.cmake

```
@@ -21,7 +21,16 @@ ExternalProject_Add(project_libbacktrace
   SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/../../3rdparty/libbacktrace
   BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/libbacktrace
   CONFIGURE_COMMAND "${CMAKE_CURRENT_LIST_DIR}/../../3rdparty/libbacktrace/configure"
-                    "--prefix=${CMAKE_CURRENT_BINARY_DIR}/libbacktrace" --with-pic
+                    "--prefix=${CMAKE_CURRENT_BINARY_DIR}/libbacktrace"
+                    --with-pic
+                    "CC=${CMAKE_C_COMPILER}"
+                    "CFLAGS=${CMAKE_C_FLAGS}"
+                    "LDFLAGS=${CMAKE_EXE_LINKER_FLAGS}"
+                    "CPP=${CMAKE_C_COMPILER} -E"
```

Maybe this should actually point to the CXX version?

```suggestion
                    "CPP=${CMAKE_CXX_COMPILER} -E"
```

---

Hi @leandron 

I think you are right, i'll do the changes and also update the documention.
On another note, i think an end to end example on inference on a remote machine would be great. 
i could probably contribute that end to end example if it does not exist, does it?

---

i looked into the github repo of  [libbacktrace](https://github.com/ianlancetaylor/libbacktrace) and it seems here ``CPP``
refers to the C preprocess (not C++), so it think it is correct to keep it as is.